Accelerated_Python_User_Guide/notebooks/Chapter_X_Distributed_Computing_cuNumeric.ipynb

```diff
@@ -233,31 +402,54 @@
   },
   {
    "cell_type": "markdown",
-   "id": "ca1739ca-cab7-4412-a345-24f1833c8f21",
+   "id": "bdd8e15e",
    "metadata": {},
    "source": [
-    "To run the following code you follow the following structure:\n",
-    "\n",
-    "legate --<processor type> --<CLI options> ./main.py <main.py options>\n",
-    "#For examples let's use:"
+    "Before running examples on Perlmutter, create an interactive session. For example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62846a5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "salloc --nodes 2 --ntasks-per-node 1 --qos interactive --time 01:30:00 --constraint gpu --gpus-per-node 4 --account=<acct_name>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b86709e3",
+   "metadata": {},
+   "source": [
+    "This command creates an allocation on 2 GPU nodes, with 4 GPUs per node. Ensure that the `legate` environment is activated. If not, load the Conda module and activate the environment. Now, run the sample program:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "de185251-38bb-47d9-a408-d7fb5412d649",
+   "id": "8131d495",
    "metadata": {},
    "outputs": [],
    "source": [
-    "legate --gpus 4 --sysmem 2000 --fbmem 12000 --eager-alloc-percentage 10 ./main.py \n"
+    "legate --gpus 4 --launcher srun --nodes 2 ./main.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca1739ca-cab7-4412-a345-24f1833c8f21",
+   "metadata": {},
+   "source": [
+    "This invocation runs legate through the `legate` driver and it executes the `main.py` example. In general, Legate can be started through Python by simply loading the appropriate modules, but in multi-node setting the driver needs to be used to enable `--launcher` and the related options.  To see more details of the resulting invocation, add `--verbose` option before `./main.py`. For more information on Command-Line Interface Options refer to the sections below or review this link and go to Section: CLI Options: [Legate CLI Options](https://docs.nvidia.com/legate/24.06/overview.html)"
```

Comment:
```suggestion
    "This invocation runs legate through the `legate` driver and it executes the `main.py` example. In general, Legate can be started through Python by simply loading the appropriate modules, but in a multi-node setting the driver needs to be used to enable `--launcher` and the related options.  To see more details of the resulting invocation, add `--verbose` option before `./main.py`. For more information on Command-Line Interface Options refer to the sections below or review this link and go to Section: CLI Options: [Legate CLI Options](https://docs.nvidia.com/legate/24.06/overview.html)"
```

Note that some recent changes have updated the links to be actual links to the appropriate documentation section.